orchagent/copporch.cpp

```diff
@@ -75,12 +78,58 @@ std::map<string, sai_packet_action_t> packet_action_map = {
     {"transit", SAI_PACKET_ACTION_TRANSIT}
 };
 
+const string default_trap_group = "default";
+
 CoppOrch::CoppOrch(DBConnector *db, string tableName) :
     Orch(db, tableName)
 {
     SWSS_LOG_ENTER();
+    initDefaultTrapGroup();
+    initDefaultTrapIds();
 };
 
+void CoppOrch::initDefaultTrapIds()
+{
+
+    sai_attribute_t attr;
+    vector<sai_attribute_t> trap_id_attrs;
+    vector<sai_hostif_trap_id_t> default_trap_ids = {
```

Comment:
move this default_trap_ids to below the const string default_trap_group as a const.